make/tools.mk

```diff
@@ -199,7 +199,7 @@ stm32flash_clean:
 DFUUTIL_DIR := $(TOOLS_DIR)/dfu-util
 
 .PHONY: dfuutil_install
-dfuutil_install: DFUUTIL_URL  := http://dfu-util.sourceforge.net/releases/dfu-util-0.8.tar.gz
+dfuutil_install: DFUUTIL_URL  := http://dfu-util.sourceforge.net/releases/dfu-util-0.9.tar.gz
```

Comment:
Really not sure we want to even offer a parasitic installation of this - users should just use the version of the tool that is provided by their distribution.
Parasitic installation is just a waste of space and a nightmare to keep up to date.

Reply:
That works!